src/dataflow/src/logging/timely.rs

```diff
@@ -294,7 +322,10 @@ pub fn construct<A: Allocate>(
 
         // Accumulate the durations of each operator.
         // The first `map` exists so that we can semijoin effectively (it requires a key-val pair).
-        let mut elapsed = duration
+        let mut elapsed: Collection<
```

Comment:
is this change required or was this just for debugging purposes?

Reply:
That's debugging cruft, oops. I'll remove it.